org.horizon_eda.HorizonEDA.json

```diff
@@ -0,0 +1,254 @@
+{
+    "app-id": "org.horizon_eda.HorizonEDA",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
+    "sdk": "org.gnome.Sdk",
+    "command": "horizon-eda",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=home",
```

Comment:
Does it require access to entire directories or projects are kept in single file? If the latter, application will automatically use a file portal if code uses GtkFileChooserNative.

Reply:
It requires access to entire directories. The user opens a project file, but the actual design data is stored in files that are in the same directory as the project file.

Reply:
Ack, thanks!